.github/workflows/ci.yml

```diff
@@ -11,16 +11,14 @@ jobs:
           node-version: '12'
       - run: npm install
       - run: sudo npm install -g knex
-      - run: knex migrate:latest --env test
-      - run: knex seed:run --env test
       - uses: paambaati/codeclimate-action@v2.4.0
         env:
           NODE_ENV: 'test'
           CC_TEST_REPORTER_ID: ${{ secrets.CodeClimateReporterId }}
-          HOSTNAME_TEST: ${{ secrets.HOSTNAME_TEST }},
-          USERNAME_TEST: ${{ secrets.USERNAME_TEST }},
-          PASSWORD_TEST: ${{ secrets.PASSWORD_TEST }},
-          PORT_TEST: ${{ secrets.PORT_TEST }},
-          NAME_TEST: ${{ secrets.NAME_TEST }},
+          HOSTNAME_TEST: 'database-1.c6akqxgwgr03.us-west-2.rds.amazonaws.com'
```

Comment:
Now you have to go and change all the secrets since this is a public facing repo 😄 
This would also be a good time opportunity to practice deleting history of this file. DM me if you need me to walk you through it.

Reply:
@michelangelo17